pepys_import/core/formats/rep_line.py

```diff
@@ -83,14 +83,14 @@ def record(self, importer, record_type, measurement_object) -> None:
         :return: Nothing
         """
 
-    def parse(self):
+    def parse(self, error, error_type):
         tokens = self.tokens()
 
         if len(tokens) < 15:
-            print(
-                "Error on line {} not enough tokens: {}".format(
-                    self.line_num, self.line
-                )
+            error.append(
```

Comment:
`error` should be `errors`, please.  It's `errors` in the calling class:
```
if not rep_line.parse(self.errors, self.message):
```

Reply:
Renamed:
https://github.com/debrief/pepys-import/pull/108/commits/363190f7324a0c27cb4c339fd717bdc115cb7819